src/components/FormFieldComponent.vue

```diff
@@ -318,8 +264,14 @@ export default {
     }
   },
   computed: {
+    id () {
+      return this.field.id
+    },
+    showError () {
+      return this.fieldState && this.fieldState.$dirty && this.fieldState.$touched && this.evaluationError[this.id]
```

Comment:
can omit this.fieldstate if you use this.fieldstate?.$dirty and this.fieldstate?.$touched

Reply:
that is a typescript thing